LeetCode/0728_Self_Dividing_Numbers.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+   def selfDividingNumbers(self, left: int, right: int) -> List[int]:
+      ans = []
+      for i in range(left,right+1):
+         flag = False
+         tempVal=i
+         while tempVal:
+               remainder = tempVal % 10
+               if not remainder:
```

Comment:
```suggestion
               if remainder == 0:
```

Reply:
actually it would be better to just do `if remainder == 0 or i % remainder != 0:`